404.html

```diff
@@ -15,12 +15,12 @@
 <!--Fallback page, don't allow indexing-->
 <meta name="robots" content="noindex">
 <h1>An open source P2P digital currency</h1>
+{% comment %}
 <p style="font-size:125%;">Bitcoin is a currency, a protocol, and a software that enables
 <ul style="font-size:125%">
-  <li>Instant peer to peer transactions</li>
   <li>Worldwide payments</li>
-  <li>Low or zero processing fees</li>
```

Comment:
Can't we just remove "or zero"? Although even now, my zero-fee transactions continue to work fine...